python_modules/dagster/dagster/_core/storage/db_io_manager.py

```diff
@@ -36,7 +44,7 @@ class TableSlice(NamedTuple):
     schema: str
     database: Optional[str] = None
     columns: Optional[Sequence[str]] = None
-    partition: Optional[TablePartition] = None
+    partition: Optional[Sequence[TablePartitionDimension]] = None
```

Comment:
I think calling this something like `partition_dimensions` would make it more clear what it contains.